addons/website_event_sale/models/sale_order.py

```diff
@@ -40,10 +40,20 @@ def _website_product_id_change(self, order_id, product_id, qty=0, **kwargs):
             values['product_id'] = ticket.product_id.id
             values['event_id'] = ticket.event_id.id
             values['event_ticket_id'] = ticket.id
+            discount = 0
             if order.pricelist_id.discount_policy == 'without_discount':
-                values['price_unit'] = ticket.price
+                if (company_currency := order.company_id.currency_id) != (pricelist_curency := order.pricelist_id.currency_id):
```

Comment:
Can we use regular variable assignation and testing please?
Not sure this `:=` operator is used anywhere within the codebase (at least not in v15).

Reply:
Yes, it was to avoid writing two lines of unnecessary code. Since the walrus operator is available in Python 3.8 and v15 uses Python 3.8, I thought it was ok to use it event if not present in the codebase.